### PR TITLE
feature/template-example-test

### LINKED ISSUE #12: Change 'make test' to 'make template_example' in on_pr_master.yml workflow file
https://github.com/fredrikhgrelland/vagrant-hashistack-template/pull/7

https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/c4dca7e92e055fb8b5843e10e470c6d1f8576596/.github/workflows/on_pr_push_master.yml#L112-L113

The above command does not trigger the code under 'template_example' and as such no tests are run so need to change it to ```make template_example```

### PR DESCRIPTION
**Pull request by zhenik**
Fix Linter rule `terraform_required_providers` , might require several changes. I would recommend do fix in separate PR

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ else
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "local_test=true"' vagrant up --provision
 endif
 
-test: clean up
+test: clean template_example
```
Comment (Neha-Sinha2305) [inline Makefile]: We will need to update the Readme as well.

**Comment by Neha-Sinha2305** [comment]
Inorder to fix the failing test jobs, we need to modify this line:
https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/c4dca7e92e055fb8b5843e10e470c6d1f8576596/.github/workflows/on_pr_push_master.yml#L110

to
```
FILE_NAME: template_example/.env_override 
```

File: Makefile (hunk 1 above)
Comment (zhenik) [reply Makefile]: outdated

File: .github/linters/.tflint.hcl [hunk 2]
```diff
@@ -22,9 +22,9 @@ rule "terraform_naming_convention" {
 rule "terraform_required_version" {
   enabled = true
 }
-rule "terraform_required_providers" {
-  enabled = true
-}
+//rule "terraform_required_providers" {
+//  enabled = true
+//}
```
Comment (Neha-Sinha2305) [inline .github/linters/.tflint.hcl:28]: This is intended change?

Comment (zhenik) [reply .github/linters/.tflint.hcl:28]: Yes, I would like to leave it here. Related to #14 we must fix it later

**Review by Neha-Sinha2305 — APPROVED** [review]
LGTM.

**Pull request merged**